utils/lolcat/Makefile

```
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 OpenWrt.org
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lolcat
+PKG_VERSION:=1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=6a2ab611a726610c43945651c9153c10ec6d8f96
+PKG_SOURCE_URL:=https://github.com/jaseg/lolcat.git
+PKG_MIRROR_HASH:=ad8c3fa6591412d4a904df1fd48c64d676c2ebb4c01d0c6d260402b2c6a72071
+
+PKG_LICENSE:=WTFPL
```

```suggestion
PKG_MAINTAINER:=
PKG_LICENSE:=WTFPL
```

You should set yourself as maintainer, btw.

---

Will do!